aqua/alacritty/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           cargo   1.0
 PortGroup           github  1.0
 
-github.setup        alacritty alacritty 0.13.1 v
+github.setup        alacritty alacritty 0.13.2 v
 github.tarball_from archive
 revision            1
```

Comment:
the "revision" field should be reset to 0 when updating the version

Reply:
Is it possible this got bumped when I ran `port lint`? I'd double checked it before committing...or so I though.

Reply:
No

Reply:
my mistake then. Sorry, fixed!